sql/org_mozilla_firefox_derived/incline_executive_v1/query.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TEMP FUNCTION bucket_manufacturer(manufacturer STRING) AS (
+  IF(
+    manufacturer IN UNNEST(['samsung', 'huawei', 'xiaomi', 'lge', 'motorola', 'sony', 'google', 'oppo', 'oneplus']),
```

Comment:
```suggestion
    manufacturer IN ('samsung', 'huawei', 'xiaomi', 'lge', 'motorola', 'sony', 'google', 'oppo', 'oneplus'),
```

I expect this simpler syntax will work.